project/app/src/main/java/com/example/cmput301w21t25/activities_experiments/ViewSubbedExperimentActivity.java

```diff
@@ -59,9 +63,13 @@ protected void onCreate(Bundle passedData) {
         TextView region = findViewById(R.id.region_text_view);
         experimentManager.FB_UpdateExperimentTextViews(expID,expName,expDesc,expType,minTrials,region);
         trialManager.FB_FetchPublishedTrialCount(exp,currTrials);
+<<<<<<< HEAD
```

Comment:
OOPS, this merge conflict resolution doesn't look right 😬

Reply:
Okay in my defense Android Studio said there was a conflict in that file but refused to tell me what it was, hold the phone